### PR TITLE
Leadersboard

### PR DESCRIPTION
**Pull request by ZiqiXu404**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 .venv
 __pycache__
+src/test.py 
+toohak-env/
+src/demo.py
```
Comment (nickbar01234) [inline .gitignore:5]: Do we need this?

**Review by tonyseetonydo — APPROVED** [review]
lgtm

**Comment by tonyseetonydo** [comment]
Steph: 👼
Us: 🙇

**Comment by nickbar01234** [comment]
@ZiqiXu404: 👼
Us: 🙇‍♂️

**Pull request merged**